### PR TITLE
Add more tests for Node (for note and wiki page titles)

### PR DESCRIPTION
**Pull request by seabl**
Fixes #0000 (no issue opened)

* [x] PR is descriptively titled 📑 
* [x] tests pass -- only Node tests
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

**Comment by seabl** [comment]
@jywarren This is a WIP. Please let me know before you merge. Thanks.

**Comment by seabl** [comment]
@publiclab/reviewers Please kindly review. Thank you.

**Comment by IshaGupta18** [comment]
Hey, could you mention the issue in the PR description please?

**Comment by IshaGupta18** [comment]
Looks good!

**Comment by seabl** [comment]
@IshaGupta18 No issue was opened for this PR. It's kind of related to #6263. Please review this while you're here. Thanks.

File: app/models/node.rb [hunk 1]
```diff
@@ -1,14 +1,12 @@
 class UniqueUrlValidator < ActiveModel::Validator
   def validate(record)
-    if record.title == '' || record.title.nil?
-      # record.errors[:base] << "You must provide a title."
+    if record.title.blank?
+      record.errors[:base] << "You must provide a title."
       # otherwise the below title uniqueness check fails, as title presence validation doesn't run until after
```
Comment (seabl) [inline app/models/node.rb]: Oddly, tests are passing whether or not the title is checked for blankness!

File: app/models/node.rb [hunk 2]
```diff
@@ -96,7 +94,7 @@ def updated_month
 
   belongs_to :user, foreign_key: 'uid'
 
-  validates :title, presence: true
+  validates :title, presence: true, length: { minimum: 3 }
```
Comment (seabl) [inline app/models/node.rb]: Anyone thinks the minimum length of the title should be longer/shorter?

File: test/unit/node_test.rb [hunk 3]
```diff
@@ -210,6 +210,41 @@ def setup
     assert_equal 'Wiki page content/body', node.body
   end
 
+  test 'wikipage with wrong title should not be created' do
```
Comment (seabl) [inline test/unit/node_test.rb]: Please compare this with the test on line 190

**Comment by seabl** [comment]
@IshaGupta18 @jywarren could you please review. Thanks!

File: app/models/node.rb (hunk 1 above)
Comment (seabl) [reply app/models/node.rb]: Is this really needed? Isn't it covered by:
`validates title, presence: true, ....` (on line 97)

**Pull request merged**